worlds/smz3/__init__.py

```diff
@@ -500,7 +500,14 @@ def modify_multidata(self, multidata: dict):
             multidata["connect_names"][new_name] = payload
 
     def fill_slot_data(self): 
-        slot_data = {}
+        slot_data = {
+            "goal": self.options.goal.value,
+            "open_tower": self.options.open_tower.value,
+            "ganon_vulnerable": self.options.ganon_vulnerable.value,
+            "open_tourian": self.options.open_tourian.value,
+            "sm_logic": self.options.sm_logic.value,
+            "key_shuffle": self.options.key_shuffle.value,
+        }
```

Comment:
There's a convenience function made for this. I think it would look something like this:
```python
        slot_data = self.options.as_dict(
            "goal",
            "open_tower",
            "ganon_vulnerable",
            "open_tourian",
            "sm_logic",
            "key_shuffle",
        )
```
(Although I wouldn't use it, because it doesn't let static analysis make sure I'm sending the right data.
But if you don't care about that, it's available.)